### DISCUSSION TITLE
Open Source Writing App

### DISCUSSION THREAD

**Discussion by ckvignesh** [EddieHubCommunity/support#3350] [General]
Hi everyone,

This is regarding an idea for open source that I have in mind...just to start a discussion and know your thoughts. I'll add the details over the next 2 weeks.

I'm an author who had trouble finding good writing software to work on my book. I agree Google Docs & MS Word are great tools but it's hard to use for writing a long book (more than 10k words).

I researched a bit and saw some non OS options like **Scrivener** and **Ulysses** (which some reputed authors use).
Their interfaces and features were awesome. I liked Scrivener's functionalities (will elaborate it later).

I explored some OS options like:
1. YWriter
2. AtomicScribbler
3. ApolloPad
4. TrueNovelist
5. Manuskript 
6. Wavemaker

I liked the last 2 tools and settled with manuskript for my book. 

But each of the existing OS options has its limitations & I thought of making a new Open Source project which solves my pain points (and of course many others' too). I'll be sharing the Pros and Cons of each tool I could find, so we can build an app which is user friendly & open source so that people can use it freely.

I'd like to build an Open Source Writing Tool for authors (technical & non-technical). All suggestions and opinions are welcome. If anyone's interested, please let me know.

**Comment by eddiejaoude** [comment]
I haven't used any of those tools. Interesting to see your review of each
